packages/components/tests/dummy/app/styles/pages/db-gutters.scss

```diff
@@ -0,0 +1,47 @@
+// GUTTERS
+
+// Loose spacing scale, move out to centralize for re-use:
+// (See: https://www.figma.com/file/oQsMzMMnynfPWpMEt91OpH/HDS-Product---Foundations?node-id=2008%3A3)
+:root {
+    --hds-spacing-xxs: 4px;
+    --hds-spacing-xs: 8px;
+    --hds-spacing-sm: 12px;
+    --hds-spacing-md: 16px;
+    --hds-spacing-lg: 24px;
+    --hds-spacing-xl: 36px;
+    --hds-spacing-xxl: 48px;
+}
+
+// TODO: Determine best default spacing size
+$default-spacing: var(--hds-spacing-md); // set as Sass variable to not expose
```

Comment:
This seemed like an appropriate default spacing size but could be changed if there's a better default to use.